hosted/Dockerfile

```diff
@@ -32,6 +31,43 @@ WORKDIR server/eb
 RUN make build-ecs
 WORKDIR /
 
+#nginx config
```

Comment:
add a comment noting why we must build from src, and that is ultimately all for the sake of supporting compression